ms2rescore/__main__.py

```diff
@@ -228,6 +228,17 @@ def main(tims=False):
         config["ms2rescore"]["log_level"], config["ms2rescore"]["output_path"] + ".log.txt"
     )
 
+    # check for software updates
+    info = check_for_update(__version__, repo="CompOmics/ms2rescore")
```

Comment:
A first note, but this is actually the case, we need to make sure that check_for_update can never fail. However, as it stands, there is no logging for errors, I would add that at least as debug logging.

Another thing though: there probably should be an option to disable this. If it were to fail, it would add a delay of 2.5 seconds, which could be one reason, but it might also be necessary in some enterprise environments where there might be regulations against these kind of outgoing requests.

Reply:
Thanks for the suggestions! Added logging to the various outcomes and a disable option in the config.